baseplate/crypto.py

```diff
@@ -46,30 +42,7 @@
 from baseplate.secrets import VersionedSecret
 
 
-if hasattr(hmac, "compare_digest"):
-    # This was added in Python 2.7.7 and 3.3
-    # pylint: disable=invalid-name,no-member
-    constant_time_compare = hmac.compare_digest
-else:
-    def constant_time_compare(actual, expected):
-        """Return whether or not two strings match.
-
-        The time taken is dependent on the number of characters provided
-        instead of the number of characters that match which makes this
-        function resistant to timing attacks.
-
-        """
-        if type(actual) != type(expected):  # pylint: disable=unidiomatic-typecheck
-            warn_deprecated("Future versions of constant_time_compare require that both "
-                "parameters are of the same type.")
-
-        actual_len = len(actual)
-        expected_len = len(expected)
-        result = actual_len ^ expected_len
-        if expected_len > 0:
-            for i in xrange(actual_len):  # noqa: F821 pylint: disable=undefined-variable
-                result |= ord(actual[i]) ^ ord(expected[i % expected_len])
-        return result == 0
+constant_time_compare = hmac.compare_digest
```

Comment:
Do we even need to keep this around?

Reply:
Probably not, but I wasn't breaking external compatibility yet.